client/src/lib/src/types/requests.ts

```diff
@@ -10,6 +10,7 @@ export enum RequestType {
   EmbeddedLanguageTypeOnPosition = 'EmbeddedLanguageTypeOnPosition',
   getLinksInDocument = 'getLinksInDocument',
   ProcessRecipeScanResults = 'ProcessRecipeScanResults',
+  ProcessGlobalEnvScanResults = 'ProcessGlobalEnvScanResults',
```

Comment:
Looks like a notification, since no result is expected (but does it make any difference?)

Reply:
Yes, by the behaviour, it should be categorized as a notification. I prefer to keep it this way just in case the client might need some response from it in the future. And as you said it doesn't make a big difference but easier to make changes afterwards.